### ISSUE TITLE
OpenLoco not opening on macOS Catalina

### ISSUE THREAD

**Issue by jonolemes** [OpenLoco/OpenLoco#392]
**OS:**  macOS 10.15
**Version:**  18.02
**Commit/Build:**  0162d4d

OpenLoco is not opening on Mac OS 10.15 Catalina due to being built as a 32bit package through the released .zip file. Mac OS 10.15 does not support 32bit Applications.

**Screenshots / Video:**
<img width="99" alt="Screen Shot 2020-01-31 at 19 56 48" src="https://user-images.githubusercontent.com/46320612/73525706-d8532480-4463-11ea-9207-a817a9051b0d.png">
<img width="435" alt="Screen Shot 2020-01-31 at 19 57 25" src="https://user-images.githubusercontent.com/46320612/73525750-e86b0400-4463-11ea-929d-4f6717bfea1b.png">

**Comment by AaronVanGeffen** [comment]
Indeed, and this is unlikely to change any time soon. The current implementation still shares a lot of routines with the original game, and as such has not yet reached a state where we can compile it as a 64-bits executable.

**Comment by joelekstrom** [comment]
@AaronVanGeffen I'd like to help out with migrating original routines if possible. Could you point me in a direction on where to look, or convince me otherwise? 😅 Eg, how do I find routines that are 32-bit dependent for example, where could one start etc?

**Comment by AaronVanGeffen** [comment]
@joelekstrom Thanks for your message. All original routines will need to be re-implemented in C++ entirely; we currently patch them in from the original game. Please join us on Gitter if you're interested: https://gitter.im/OpenLoco/OpenLoco
